9.4.Методы_строк_Часть_2/9.4.11.py

```diff
@@ -0,0 +1,7 @@
+n = input()
+count = 0  
+
+for i in range(10):
+    count += n.count(str(i))
```

Comment:
Интересное решение, но оно контринтуитивно. То есть ты задаешь последовательность от 0 до 9 и добавляешь к кол-ву цифр в строке кол-во цифр из последовательности.

Это прям сложно и лучше не изобретать велосипеды и воспользоваться методом `.isdigit()`, который проверит символ, является ли он цифрой:
```suggestion
for char in n:
    if char.isdigit():
        count += 1
```